menas/ui/components/dataset/conformanceRule/ConformanceRuleForm.js

```diff
@@ -386,9 +395,18 @@ class MappingConformanceRuleForm extends ConformanceRuleForm {
     return isValid
   }
 
+  hasValidOutputColumns(rule = []) {
```

Comment:
Since the validation is done in `OutputColumnDialog`. `hasValidOutputColumns` cannot be accessed in `ConformanceRuleForm`, because it's now only related to Mapping rule so it needs to be called differently